src/components/settings/owner/ChangeThresholdDialog/index.tsx

```diff
@@ -90,27 +91,24 @@ const ChangeThresholdStep = ({ data, onSubmit }: { data: ChangeThresholdData; on
           </Grid>
         </Grid>
 
-        {!selectedThreshold && (
-          <Typography mb={2}>
-            Current policy is{' '}
-            <b>
-              {data.threshold} out of {safe.owners.length}
-            </b>
-            .
-          </Typography>
-        )}
+        <Typography mb={2}>
+          Current policy is{' '}
+          <b>
+            {data.threshold} out of {safe.owners.length}
+          </b>
+          .
+        </Typography>
       </DialogContent>
 
-      {selectedThreshold && (
-        <Box mt={-5}>
-          <SignOrExecuteForm
-            safeTx={safeTx}
-            isExecutable={safe.threshold === 1}
-            onSubmit={onChangeTheshold}
-            error={safeTxError}
-          />
-        </Box>
-      )}
+      <Box mt={-5}>
+        <SignOrExecuteForm
+          safeTx={safeTx}
+          isInitiallySubmittable={selectedThreshold !== data.threshold}
```

Comment:
Can't we leave it as submittable but exit or throw on submit if the form is invalid?